runtime/default.policy

```diff
@@ -956,7 +961,7 @@ rules:
       (process.parent.file.name in WEBAPP_PROCESSES || process.parent.file.name =~ "php*")
     filters:
       - os == "linux"
-  - id: procdump_usage
+  - id: procdump_execution
```

Comment:
are those id changes in sync with backend rules ?

Reply:
yep!

Reply:
https://github.com/DataDog/security-monitoring/blob/main/workload-security/backend-rules/procdump_usage.yaml